cu-webteam/d8-platform
# News displays

# Description

This adds styling, templates and configuration for news node display and news views.

## Related PRs

N/A

## Todos

- [X ] Tested on local machine with visual test by Beth Cavanaugh
- [ ] Documentation (none at this time)

## Deploy Notes

N/A

## Steps to Test or Reproduce

Create news content using the News/Spotlight content type.
Load in [siteurl]/news to see the view
If a story does NOT have a news link, click on the headline in the view to view node.
Otherwise, go to [siteurl]/admin/content and click on the title of the news to view the node.
